ocd_backend/models/model.py

```diff
@@ -218,6 +220,13 @@ def save(self):
         finally:
             self.saving_flag = False
 
+    def merge(self, **kwargs):
```

Comment:
As discussed, calls to `merge` can be implicit in most cases. Only needs to be explicit when we have two different sources like CBS and Allmanak, since they will never have the same `had_primary_source`.

Reply:
Suggestion for the method name, but still doesn't fully covers it: `set_ori_identifier`

Reply:
@fletcher91 can you come up with a name?